nightmare_firmware/nightmare_firmware.ino

```diff
@@ -22,6 +24,7 @@ void setup(){
   	digitalWrite(FAN2_PIN, LOW);
 
 	servo::init(SERVO_PIN_TX_ENB, SERVO_PIN_RX_ENB); //init servo circuitry
+	hservo.setPeriodHertz(50);
```

Comment:
maybe would be better to have a const somewhere

Reply:
fixed in upcoming push